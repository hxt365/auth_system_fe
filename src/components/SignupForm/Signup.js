import React from 'react';

function SignupForm() {
  return <p>SignupForm</p>;
}

export default SignupForm;
