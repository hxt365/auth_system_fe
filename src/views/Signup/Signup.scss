.signup {
  text-align: center;
  padding-top: 5rem;
}
