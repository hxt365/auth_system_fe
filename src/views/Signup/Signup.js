import React from 'react';
import './Signup.scss';
import SignupForm from 'components/SignupForm';
import { Link } from 'react-router-dom';
import { LOGIN } from 'constants/index';

function Signup() {
  return (
    <section className="signup">
      <Link to={LOGIN} className="signup__link">
        Already have account?
      </Link>
      &nbsp;Or
      <h2>Just a minute..</h2>
      <SignupForm />
    </section>
  );
}

export default Signup;
