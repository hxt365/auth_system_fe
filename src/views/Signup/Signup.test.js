import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Signup from './Signup';

describe('test Signup', () => {
  it('test snapshot', () => {
    const wrapper = shallow(<Signup />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
