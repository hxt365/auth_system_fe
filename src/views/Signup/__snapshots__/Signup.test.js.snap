// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Signup test snapshot 1`] = `
<section
  className="signup"
>
  <Link
    className="signup__link"
    to="/login"
  >
    Already have account?
  </Link>
  Â Or
  <h2>
    Just a minute..
  </h2>
  <Form(SignupForm) />
</section>
`;
